cmd/substreams/run.go

```diff
@@ -32,19 +34,31 @@ func init() {
 
 // runCmd represents the command to run substreams remotely
 var runCmd = &cobra.Command{
-	Use:          "run <manifest> <module_name>",
-	Short:        "Stream module outputs from a given package on a remote endpoint",
+	Use:   "run [<manifest>] <module_name>",
+	Short: "Stream module outputs from a given package on a remote endpoint",
+	Long: cli.Dedent(`
+		Stream module outputs from a given package on a remote endpoint. The manifest is optional as it will try to find 
+		one in your pwd if nothing entered. You may enter a dir that contains a 'substreams.yaml' file in place of <manifest_file>
+	`),
 	RunE:         runRun,
-	Args:         cobra.ExactArgs(2),
+	Args:         cobra.RangeArgs(1, 2),
 	SilenceUsage: true,
 }
 
 func runRun(cmd *cobra.Command, args []string) error {
 	ctx := cmd.Context()
-
 	outputMode := mustGetString(cmd, "output")
 
-	manifestPath := args[0]
+	manifestPathRaw := ""
+
+	if len(args) == 2 {
+		manifestPathRaw = args[0]
+		args = args[1:]
+	}
+	manifestPath, err := tools.ResolveManifestFile(manifestPathRaw)
+	if err != nil {
+		return fmt.Errorf("resolving manifest: %w", err)
+	}
```

Comment:
What is the current behavior on the `run` command if the user makes the mistake of forgetting to put the `<module>` so `substreams run substreams.yaml -e mainnet....`. I would like that we improve a bit this experience event if maybe rare.

We can be a little more clever, if there is a single argument and it contains a '.' or it's a valid directory, we know we are missing the output module arg and we can improve the improve the error message.

We have helper `cli.DirectoryExists(path string)` that you can quickly use to determine if the input is a directory (returns false on 'error' which is good enough for our use case)